### PR TITLE
Update odata-connection-manager.md

### PR DESCRIPTION
**Pull request by KrishnakumarRukmangathan**
Added a known localization limitation of OData connection manager where the VS/SSDT language settings needs to aligned to the target SQL Server where the SSIS packages are deployed.

### CODE REVIEW

**Comment by PRMerger5** [comment]
@KrishnakumarRukmangathan : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by chugugrace — APPROVED** [review]

**Comment by chugugrace** [comment]
#sign-off

File: docs/integration-services/connection-manager/odata-connection-manager.md [hunk 1]
```diff
@@ -30,7 +30,9 @@ ms.author: chugu
 -   Right-click the **Connection Managers** folder in **Solution Explorer**, and then click **New Connection Manager**. Select **ODATA** for **Connection manager type**.  
   
 -   Right-click in the **Connection Managers** pane at the bottom of the package designer, and then select **New Connection**. Select **ODATA** for **Connection manager type**.  
-  
+> [!NOTE]
+> Make sure that international settings language in your Visual Studio / SQL Server Data Tools (SSDT) is aligned to the default language of the target SQL Server where the SSIS packages are deployed.
```
Comment (ShannonLeavitt) [inline docs/integration-services/connection-manager/odata-connection-manager.md:34]:
```suggestion
> Make sure that the international settings language in your Visual Studio / SQL Server Data Tools (SSDT) is aligned to the default language of the target SQL Server instance where the SSIS packages are deployed.
```
Fixing language and terminology

**Comment by PRMerger4** [comment]
@ShannonLeavitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**